android/src/paper/java/com/facebook/react/viewmanagers/RNSScreenManagerDelegate.java

```diff
@@ -91,7 +91,7 @@ public void setProperty(T view, String propName, @Nullable Object value) {
         mViewManager.setStackAnimation(view, (String) value);
         break;
       case "transitionDuration":
-        mViewManager.setTransitionDuration(view, value == null ? 350 : ((Double) value).intValue());
+        mViewManager.setTransitionDuration(view, value == null ? 500 : ((Double) value).intValue());
```

Comment:
Out of curiosity, why we want to have `500ms` there? Is it taken from your own observations, or is it supported by iOS documentation?

Reply:
@tboba I would say both 😅 I measured duration of animation in `QuickTime` player and it was showing ~480ms (which is very close to 500ms and 20ms could be due to incorrect cut out of a video).
And second observation is that `7 << 16` animation takes 500ms to complete 🙃